requirements.txt

```diff
@@ -643,7 +643,7 @@ regex==2024.11.6 \
     --hash=sha256:fdabbfc59f2c6edba2a6622c647b716e34e8e3867e0ab975412c5c2f79b82da2 \
     --hash=sha256:fdd6028445d2460f33136c55eeb1f601ab06d74cb3347132e1c24250187500d9 \
     --hash=sha256:ff590880083d60acc0433f9c3f713c51f7ac6ebb9adf889c79a261ecf541aa91
-requests==2.32.3 \
+requests>=2.31.0 \
```

Comment:
we should **never** edit this requirement.txt manually. 

**always do:**
```bash
uv sync
uv add pymupdf4llm fitz <other new dependencies>
uv export --format requirements-txt > requirements.txt

```

Reply:
@arshiaesll i've pushed a [fix](https://github.com/nazdridoy/kokoro-tts/pull/18/commits/adac17d28fa5f827ceef7eef18fe4f73563e2ad0)